creating_packages/existing_binaries.rst

```diff
@@ -0,0 +1,123 @@
+.. _existing_binaries:
+
+Packaging existing binaries
+===========================
+
+Sometimes, it is necessary to create packages from existing binaries, like binaries from third
+parties, or previously built by another process or team not using conan, so building from sources is
+not wanted. You would want to package local files in two situations:
+
+ - When it is not possible to build the packages from sources (only pre-built binaries available).
+ - When you are developing your package locally and want to export the built artifacts to the local
+   cache.
+   As you don't want to rebuild again (clean copy) your artifacts, you don't want to call
+   **$ conan create**.
```

Comment:
Regarding format, I am not sure the bold is the best for command line examples.  I am fine with using a different one, like italics for files/folders.

Reply:
yes, not sure about bold for command line commands but I couldn't find other option, that's why I used the $